core/src/main/java/ch/cyberduck/core/Resolver.java

```diff
@@ -73,9 +63,6 @@ public void run() {
                 try {
                     final InetAddress[] allByName = InetAddress.getAllByName(hostname);
                     Arrays.stream(allByName).findFirst().ifPresent(resolved::set);
-                    if(preferIPv6) {
```

Comment:
it shouldn't - because this is about preferring IPv6 addresses over IPv4 - not forcing connections exclusively to IPv6.

The code already uses the `getAllByName()`  that @ylangisc has referenced in the issue report.

Reply:
This is correct, given we would set the default of `java.net.preferIPv6Addresses` to `true`, no additional sorting should be required for IPv6 addresses to be preferred.